en/core-api/bank-transfer.md

```diff
@@ -1,69 +1,62 @@
 # Core API Bank Transfer Integration
 <hr>
-Basic integration process of Bank Transfer (Virtual Account) will be explained below.
-
-?> Please make sure you have already done [creating your Midtrans Account](/en/midtrans_account/overview), before proceeding with this section.
-
-One of the payment method offered by Midtrans is Bank Transfer. By using this payment method, customers will have the option to make a payment via bank transfer and Midtrans will send real time notification when the customer complete the payment.
-
-At this moment, Midtrans has integrated with the following bank transfer payment methods:
-
-1. BCA Virtual Account
-2. BNI Virtual Account
-3. BRI Virtual Account
-4. Mandiri Bill Payment
-5. Permata Virtual Account
-
-## Integration Step
-1. Send transaction data to API Charge.
-2. Display virtual account number and expiry time.
-3. Handling Post-Transaction.
-
+Basic integration process for Bank Transfer (Virtual Account) is explained in this section.
+Your customers can make payments using the <i>Bank Transfer</i> payment method provided by Midtrans. You will be notified when customer completes the transaction using this option. A list of banks supported by Midtrans is given below.
+<ol><li>BCA Virtual Account</li>
+    <li>BNI Virtual Account</li>
+    <li>BRI Virtual Account</li>
+    <li>Mandiri Bill Payment</li>
+    <li>Permata Virtual Account</li></ol>
+For each respective bank, Midtrans creates a virtual account which is allocated to merchant.
+
+?> Please make sure to create your [Midtrans account](/en/midtrans-account/overview), before proceeding with this section.
 <details>
 <summary><b>Sequence Diagram</b></summary>
 <article>
-The overall Bank Tranfer end-to-end payment proccess can be illustrated in following sequence diagram:
-
+The overall Bank Transfer end-to-end payment process is illustrated in following sequence diagram:
 ![bank transfer sequence diagram](./../../asset/image/core_api-sequence_bank_transfer.png)
 </article>
 </details>
 
-Charge API request should be done from Merchant's backend. Server Key (from your account's Dashboard) will be needed to [authenticate the request](https://api-docs.midtrans.com/#http-s-header).
+## Sandbox Environment
+The steps given below use [Midtrans *Sandbox* environment](https://account.midtrans.com/) to test the integration process. Please make sure that you use the *Server Key* and *Client Key* for the *Sandbox* environment. For more details, refer to [Retrieving API Access Keys](/en/midtrans-account/overview.md#retrieving-api-access-keys).
+##  Steps for Integration
+To integrate with Bank Transfer payment method, follow the steps given below.
+#### 1. Sending Transaction Data to API Charge
+Charge API request should be generated from your backend. The request is authenticated with a server key, which can be accessed through the account. After the request is sent, you will get the va_number.
```

Comment:
Important keyword like `va_number` not getting highlighted? this is a bit inconsistent